### ISSUE TITLE
Aki.Launcher.exe refuse to run EscapeFromTarkov.exe if *.bak file in user\mods dir.

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#741]
SPT-AKI 3.7.5 Release.

1. Copy order.json to order.bak
2. Try to run Tarkov.

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/213) by _**Barlog_M**_

**Comment by refringe** [comment]
this seems unintentional, I'm looking into it.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/213#issuecomment-20752) by _**waffle.lord**_

**Comment by refringe** [comment]
so from what I can tell, we've made the decision to restore all .bak files before starting the game to ensure any patches that need to be applied are done to the original files.


the error above was due to not catching an exception from a .bak file resolving to a directory or non-existing file.

- "something.bak" resolves to directory "something" which isn't valid
- "something.json.bak" resolves to a non-existent file "something.json"


These exceptions are now logged and should be fixed in a future release.


Be aware that if you are creating any .bak files in your game directory, they may be restored at launch if they resolve to a valid file

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/213#issuecomment-20755) by _**waffle.lord**_
